CMakeLists.txt

```diff
@@ -941,4 +941,5 @@ install(TARGETS shadps4 BUNDLE DESTINATION .)
 if (ENABLE_QT_GUI AND CMAKE_SYSTEM_NAME STREQUAL "Linux")
     install(FILES ".github/shadps4.desktop" DESTINATION "share/applications")
     install(FILES ".github/shadps4.png" DESTINATION "share/icons/hicolor/512x512/apps")
```

Comment:
The 512x512 icon should have the same name as the scalable one.

Reply:
https://github.com/shadps4-emu/shadPS4/pull/1495 includes a change to make that happen :)